src/printer.js

```diff
@@ -1920,10 +1925,10 @@ function printArgumentsList(path, options, print) {
           lastArg.body.type === "ObjectExpression" ||
           lastArg.body.type === "ArrayExpression" ||
           lastArg.body.type === "CallExpression" ||
-         lastArg.body.type === "JSXElement"))) &&
-        // If the last two arguments are of the same type,
-        // disable last element expansion.
-        (!penultimateArg || penultimateArg.type !== lastArg.type);
```

Comment:
It's so nice not to have to care about those kind of things during review and know that it'll be correctly formatted eventually :)